FlowCrypt/Functionality/Error Handling/Task+Retry.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Task+Retry.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 3/30/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+extension Task where Failure == Error {
+    @discardableResult
+    static func retrying(
+        priority: TaskPriority? = nil,
+        maxRetryCount: Int = 3,
+        retryDelay: TimeInterval = 1,
+        operation: @Sendable @escaping () async throws -> Success
+    ) -> Task {
+        Task(priority: priority) {
+            for _ in 0 ..< maxRetryCount {
+                do {
+                    return try await operation()
+                } catch {
+                    let oneSecond = TimeInterval(1_000_000_000)
+                    let delay = UInt64(oneSecond * retryDelay)
+                    try await Task<Never, Never>.sleep(nanoseconds: delay)
+
+                    continue
```

Comment:
If you are at the end of a loop block, I don't think `continue` would have any effect.

Reply:
Right. I added `continue` to make intent more clear.
Do you want me to remove that?

Reply:
yes - please remove

Reply:
OK